seed/go-sdk/imdb/no-custom-config/README.md

```diff
@@ -4,6 +4,44 @@
 
 The Seed Go library provides convenient access to the Seed API from Go.
 
+## Advanced
+
+### Retries
+
+The SDK is instrumented with automatic retries with exponential backoff. A request will be retried as long
+as the request is deemed retryable and the number of retry attempts has not grown larger than the configured
+retry limit (default: 2).
+
+A request is deemed retryable when any of the following HTTP status codes is returned:
+
+- [408](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/408) (Timeout)
+- [429](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/429) (Too Many Requests)
+- [5XX](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/500) (Internal Server Errors)
+
+Use the `option.WithMaxAttempts` option to configure this behavior for the entire client or an individual request:
+
+```go
+client := seedclient.NewClient(
+    option.WithMaxAttempts(1),
+)
+
+response, err := client.Imdb.CreateMovie(
+    ...,
+    option.WithMaxAttempts(1),
+)
+```
+
+### Timeouts
+
+Setting a timeout for each individual request is as simple as using the standard context library. Setting a one second timeout for an individual API call looks like the following:
+
+```go
+ctx, cancel := context.WithTimeout(..., time.Second)
+defer cancel()
+
+response, err := client.Imdb.CreateMovie(...)
```

Comment:
nit: For this one, let's go ahead an reuse the `ctx` variable naming in the `context.WithTimeout` call - it's a little more clear to the user that a `context.Contet` variable is expected, and it's [still valid Go code](https://go.dev/play/p/1WmlJ3EALUx). We can also show that it's used in the next invocation call.

```go
ctx, cancel := context.WithTimeout(ctx, time.Second)
defer cancel()

response, err := client.Imdb.CreateMovie(ctx, ...)
```